composer.json

```diff
@@ -24,7 +24,7 @@
   "require": {
     "php": ">=7.0",
     "fei/connect-common": "^3.0.0",
-    "fei/api-client": "~1.2.0",
+    "fei/api-client": "^1.3",
```

Comment:
@mtisserant Have you test the client after this upgrade ?

Reply:
this upgrade add constant on AbstractApiClient class. it's not a breaking change. I don't test it on local env yet.